test/test-framework/.osdk-scorecard-bundle.yaml

```diff
@@ -0,0 +1,11 @@
+scorecard:
+  version: v1alpha2
+  bundle: deploy/olm-catalog/memcached-operator
+  plugins:
+    - olm:
+        cr-manifest:
+          - "deploy/crds/cache.example.com_v1alpha1_memcached_cr.yaml"
+        init-timeout: 60
+        csv-path: "deploy/olm-catalog/memcached-operator/0.0.3/memcached-operator.v0.0.3.clusterserviceversion.yaml"
+        proxy-image: "quay.io/example/scorecard-proxy"
+        proxy-pull-policy: "Never"
```

Comment:
Now, I am confused. The bundle is not the OLM bundle, it is the scorecard config YAML. Am I right?
So, I do not think that we can use the name bundle since it is adopted nomenclature to define the OLM bundle required to publish an operator. See:  https://github.com/operator-framework/operator-registry#manifest-format

<img width="1046" alt="Screenshot 2019-12-12 at 15 47 16" src="https://user-images.githubusercontent.com/7708031/70726901-b1dffb00-1cf6-11ea-923f-d234cb18db25.png">

Reply:
Why we need some many diff layouts of the files for the scorecard?
What is the diff between https://github.com/operator-framework/operator-sdk/blob/7111c0cc25b15e301b1d0668640981dd062dfc6e/test/test-framework/.osdk-scorecard-bundle.yaml and https://github.com/operator-framework/operator-sdk/blob/7111c0cc25b15e301b1d0668640981dd062dfc6e/test/test-framework/.osdk-scorecard-bundle.yaml?

Reply:
the .osdk-scorecard-bundle.yaml file specifies the new bundle flag, the other versions of this config are left as-is...I think there is a refactor story to re-do configuration and testing in general for the scorecard.

Reply:
But the name bundle lead to me understand that the dir will have the CSV and CRD's files which is not the case.I think we need re-check/change the nomenclature in order to avoid misunderstandings.

Reply:
here I'm referring to an OLM bundle, the validation API expects that...I've been testing this with an OLM generated bundle manifest like under ~/projects/memcached-operator/deploy/olm-catalog/memcached-operator.  That directory does have a CSV and CRD in its tree.  Also a package manifest is checked by this API.

Reply:
Oh, great tks. It was confusing. So all sort out